src/components/SplitHorizontal/SplitHorizontal.jsx

```diff
@@ -387,6 +387,7 @@ const SplitHorizontal = createClass({
 					'&-is-animated': isAnimated,
 				}, className)}
 				style={{
+					display: 'flex',
```

Comment:
yeah looks good to me, not sure how i forgot to give both panes `overflow: 'auto'`, so this is a good catch!